### PR TITLE
hotplug: wait for device initialization

### PR DESCRIPTION
**Pull request by yarda**
This is the first shot trying to resolve the potential problem with hotplug. It needs thorough testing. I am not sure whether it's a good thing to use it also for the remove events, maybe it would require refactor.

### CODE REVIEW

File: tuned/hardware/inventory.py [hunk 1]
```diff
@@ -54,6 +54,14 @@ def _handle_udev_event(self, event, device):
 		if not device.subsystem in self._subscriptions:
 			return
 
+		retry = consts.HOTPLUG_WAIT_FOR_DEV_INIT_RETRIES
+		while not device.is_initialized and retry > 0:
+			log.debug("Device '%s' is uninitialized, waiting '%d' seconds for its initialization." % (device, consts.HOTPLUG_WAIT_FOR_DEV_INIT_DELAY))
```
Comment (zacikpa) [inline tuned/hardware/inventory.py:59]: Maybe print this as float, the current log would just say 0 seconds in the default configuration.
```suggestion
			log.debug("Device '%s' is uninitialized, waiting '%.2f' seconds for its initialization." % (device, consts.HOTPLUG_WAIT_FOR_DEV_INIT_DELAY))
```

**Review by zacikpa — COMMENTED** [review]
The changes look good to me, but I did no thorough testing. Just one nit:

File: tuned/hardware/inventory.py (hunk 1 above)
Comment (yarda) [reply tuned/hardware/inventory.py]: Thanks, fixed.

**Review by zacikpa — APPROVED** [review]

**Comment by yarda** [comment]
Centos 7 CI failure is caused by dropped EPEL-7, it's being worked on.

**Pull request merged**